ElectronClient/app.js

```diff
@@ -232,6 +233,16 @@ class Application extends BaseApplication {
 				newState.devToolsVisible = action.value;
 				break;
 
+			case 'THEME_TOGGLE':
+				newState = Object.assign({}, state);
+				newState.isNightMode = !state.isNightMode;
+				break;
+
+			case 'THEME_TOGGLE_SET':
```

Comment:
Probably better named "NIGHT_MODE_SET" as it is not actually a toggle.

Reply:
Oops! I forgot to update name before committing to repo. Before NIGHT MODE Button I named THEME button when I was doing trial run.. I'll update the name.